oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/meter/config/MeterConfig.java

```diff
@@ -29,8 +30,16 @@
 public class MeterConfig implements MetricRuleConfig {
     private String metricPrefix;
     private String expSuffix;
+    private String expPrefix;
     private String filter;
     private List<Rule> metricsRules;
+    private boolean kubernetesAware;
```

Comment:
How about adding an initExp? And put k8s listening function inside it?

In my understanding, MAL should be not environment sensitive. The functions should do the logic things. MAL is only a skeleton.